charts/cosigned/README.md

```diff
@@ -0,0 +1,90 @@
+# Cosigned Admission Webhook
+
+## Requirements
+* kind (or any other Kubernetes cluster).
+* Helm.
+
+## Run `cosigned`
+
+Cosigned requires `cert-manager` to be pre-configured on the running cluster.
+To install `cert-manager` follow the next steps:
+
+```shell
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.5.0 \
+  --set installCRDs=true
+```
+
+Once `cert-manager` is installed in your cluster, you can start configuring `cosigned`.
+
+Generate a keypair to validate the signatures of the deployed Kubernetes resources and their images:
+
+```shell
```

Comment:
we can use the following commands to store key pairs in Kubernetes secret, you can get more detail about it from [here](https://github.com/sigstore/cosign/blob/main/KMS.md#kubernetes-secret).
```shell
# generate a key-pair in Kubernetes Secret
cosign generate-key-pair k8s://[NAMESPACE]/[NAME]
```

so, the user does not have to create a secret manually to store these. 🙏

Reply:
This is another thing to improve, i just let it as it was from the original code. But it definitely needs a re-design or re-think this approach so it can works with all the supported options (e.g. KMS, ...)